frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimesView.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import ApplicationsPage from '~/pages/ApplicationsPage';
+import EmptyCustomServingRuntime from './EmptyCustomServingRuntime';
+import { CustomServingRuntimeContext } from './CustomServingRuntimeContext';
+
+const CustomServingRuntimesView: React.FC = () => {
+  const {
+    servingRuntimeTemplates: { data: servingRuntimeTemplates },
+  } = React.useContext(CustomServingRuntimeContext);
+
+  return (
+    <ApplicationsPage
+      title="Serving runtimes"
+      description="Manage model serving runtimes"
+      loaded // already checked this in the context provider so loaded is always true here
```

Comment:
Question you should ask yourself is -- is there a reason for the spinner in the Context.

Reply:
I would think so, just to have uniformity between the other contexts, they all have this same flow, maybe I can remove the comment since it can be redundant.

Reply:
And the advantage of having the loader in the context rather than in the page is that in router pages, no matter where you are you'll be sure to display an error page/loading error rather than having to implement that in every screen.

Reply:
Fair enough -- lets go with this and swing back later then. You probably don't need the loaded comment... but I see no harm in leaving it if you want.